src/index.ts

```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
 }`;
 }
 
+function addComment(crawlerId: string): void {
+  try {
+    const pathArray = CRAWLER_API_BASE_URL.split('/');
+    const protocol = pathArray[0];
+    const host = pathArray[2];
+    const baseUrl = `${protocol}//${host}`;
+
+    const message = `<p>Check Crawler: ${baseUrl}/admin/crawlers/${crawlerId}/overview</p>
+    <p>Check Application: https://www.algolia.com/apps/${ALGOLIA_APP_ID}/explorer/browse/${CRAWLER_NAME}</p>`;
+
+    const context = github.context;
+    if (context.payload.pull_request === undefined) {
+      core.setFailed('No pull request found.');
```

Comment:
It will fail on master, I don't think comment is a mandatory step, it should just log.